pkg/registry/registry.go

```diff
@@ -93,12 +123,19 @@ func (r *registry) LastLayer(ctx context.Context, entry string) (v1.Layer, error
 	if hash := strings.Split(entry, "@"); len(hash) > 1 {
 		repo = hash[0]
 	} else if tag := strings.Split(entry, ":"); len(tag) > 1 {
+
 		repo = tag[0]
 	}
 
-	manifest, err := crane.Manifest(entry)
+	var registryAuths []crane.Option
+	if auth.Email != "" && auth.Auth != "" {
+		registryAuths = append(registryAuths, crane.WithAuth(authn.FromConfig(authn.AuthConfig{Username: auth.Email, Auth: auth.Auth})))
+	}
+
+	manifest, err := crane.Manifest(entry, registryAuths...)
 	if err != nil {
-		return nil, err
+		utils.WarnOnError(fmt.Errorf("cannot extract manifest: %v", err))
+		return nil, nil
```

Comment:
This was recently changed to return an error, it prevents segfaults.

Reply:
Oops, sorry. Thanks for catching!